Shopify/shopify-cli
# Bump javy version to 0.2.0

### WHY are these changes introduced?
We have a new javy release that will compile the scripts to a wasm that is compatible with the new spec https://github.com/Shopify/javy/releases/tag/v0.2.0

Closes https://github.com/Shopify/script-service/issues/3855

### How to test your changes?
1. Run this branch's javy on built `index.js` from a script project `shopify-dev script javy --in index.js --out script.wasm`
2. Convert the wasm to a `.wat` file https://webassembly.github.io/wabt/demo/wasm2wat/
3. Verify that the `.wat` exports ` _start` 
Something like this
````
  (export "_start" (func 1293))
````

### Update checklist
